lib/screens/action/action_flow.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:provenance_dart/wallet_connect.dart';
+import 'package:provenance_wallet/common/flow_base.dart';
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/modal/pw_modal_screen.dart';
+import 'package:provenance_wallet/screens/action/action_list/action_list_bloc.dart';
+import 'package:provenance_wallet/screens/action/action_list/action_list_screen.dart';
+import 'package:provenance_wallet/screens/transaction/transaction_confirm_screen.dart';
+import 'package:provenance_wallet/services/models/account.dart';
+import 'package:provenance_wallet/services/models/requests/send_request.dart';
+import 'package:provenance_wallet/services/models/requests/sign_request.dart';
+import 'package:provenance_wallet/services/models/wallet_connect_session_request_data.dart';
+import 'package:provenance_wallet/util/assets.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/messages/message_field_name.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ActionFlow extends FlowBase {
+  const ActionFlow({required this.account, Key? key}) : super(key: key);
+
+  final Account account;
+
+  @override
+  State<StatefulWidget> createState() => ActionFlowState();
+}
+
+class ActionFlowState extends FlowBaseState implements ActionListNavigator {
+  @override
+  void initState() {
+    super.initState();
+
+    get.pushNewScope(scopeName: "ActionScope");
+  }
+
+  @override
+  void dispose() {
+    get.popScope();
```

Comment:
Does this for sure remove a named scope?

Reply:
yes

Reply:
dont think of it as a named scope. It is a new scope that overlays the previous. The name is for debugging purposes